Settings file isn't being read by MinecraftLauncher class

Symptom - Selected mod pack always launches in full screen.

[Console log or GTFO](http://pastebin.com/yjRba3va)
Error is specifically at line 72, but happens anytime the window is resized.
"[ERROR] FileOutputStream.open:-2->Settings.save:55: Failed to save settings: java.io.FileNotFoundException: /Users/nickfajardo/ftb/Direwolf20/Library/Application Support/ftblauncher/ftblaunch.cfg (No such file or directory)"

Cause - 
System property user.home is being reset inside the MinecraftLauncher class [MinecraftLauncher.java#L152](https://github.com/Slowpoke101/FTBLaunch/blob/master/src/net/ftb/mclauncher/MinecraftLauncher.java#L152) so the FTB config file isn't being found.  For OS X, user.home is normally /Users/<username>.   This is likely also an issue on *nix systems, as the config file there also uses user.home property

Lookup of Settings file [Settings.java#L33](https://github.com/Slowpoke101/FTBLaunch/blob/master/src/net/ftb/data/Settings.java#L33)
OSUtils.getDynamicStorageLocation() is defined here [OSUtils.java#L51](https://github.com/Slowpoke101/FTBLaunch/blob/master/src/net/ftb/util/OSUtils.java#L51)

Possible solutions
- Revert changes in https://github.com/Slowpoke101/FTBLaunch/commit/f6cbdff46f4ef8384191eb77a65d7f7e588d938a
- Investigate to see if user.home system property actually needs to be changed inside MinecraftLauncher
- Pass Settings file location to MinecraftLauncher.java in the arguments.  Create overload constructor on Settings.java to allow the different file.  Have MinecraftFrame use this other instance of the Settings class, instead of Settings.getSettings()

--- a/src/net/ftb/mclauncher/MinecraftLauncher.java
+++ b/src/net/ftb/mclauncher/MinecraftLauncher.java
@@ -107,6 +107,7 @@ public class MinecraftLauncher {
 
 	public static void main(String[] args) {
 		String basepath = args[0], animationname = args[1], forgename = args[2], username = args[3], password = args[4], modPackName = args[5], modPackImageName = args[6];
+		Settings.getSettings().save();  //Call so that the settings file is loaded from the correct location.  Would be wrong on OS X and *nix if called after user.home is reset
 
 		try {
 			System.out.println("Loading jars...");

